lib/spack/env/cc

```diff
@@ -145,6 +153,11 @@ fi
 # Save original command for debug logging
 input_command="$@"
 
+if [ "$mode" == vcheck ] ; then
+    exec "${input_command}"
+    exit
```

Comment:
You can leave out the `exit`, or if you want to keep it, you should exit with an error instead.